de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -135,7 +145,13 @@ public void tearDown() throws CoreException {
 					
 					// Now reset the workbench and remove the project
 					Activator.getDefault().getLog().log(new Status(Status.INFO, Activator.getPluginId(), "ASwtBotTestCase: Resetting Workbench"));
-					bot.resetWorkbench();
+					
+					// bot.resetWorkbench(); BUG: Resetting the workbench this way causes problems with the UI palette during tearDown					

```

Comment:
Maybe we can be a bit more specific here.
I think it would be interesting to know that the issue is in the UI palette of graphiti and according to your investigations I believe it was related to some shells being disposed?